### ISSUE TITLE
Issues

### ISSUE THREAD

**Issue by Macumbafeh** [fuba82/fuba_DepositBank-TBC#1]
Hello i don't know if this addon is finished but i got thoses errors. and the addon don't seems to work

![deposit](https://user-images.githubusercontent.com/47739411/93453483-080f4980-f8da-11ea-89e5-68e4b6fff26d.jpg)
![deposit2](https://user-images.githubusercontent.com/47739411/93453495-0a71a380-f8da-11ea-8520-dd03876dcf39.jpg)
![deposit3](https://user-images.githubusercontent.com/47739411/93453503-0e052a80-f8da-11ea-8413-c0c2810bcc66.jpg)

thanks.

**Comment by Macumbafeh** [comment]
![image](https://user-images.githubusercontent.com/47739411/93454048-b87d4d80-f8da-11ea-9bc5-2ec678bbe69a.png)
![image](https://user-images.githubusercontent.com/47739411/93454077-c206b580-f8da-11ea-8b94-34f32b744357.png)

**Comment by fubaWoW** [comment]
oh yeah it is not updated on github becasue it's not released :D
i will update it then

**Comment by Macumbafeh** [comment]
Thanks a lot. Let me know when you do it.
You Can find us on my discord, i'm the admin of TBC addon 2.4.3. all addons for TBC :)
https://discord.gg/5qVu56M

**Comment by fuba82** [comment]
AHH  
found the issue :D  

This AddOn also requires a **Compatibility** AddOn i've not recognized on release...  
I've Uploaded the [Compatibility-Addon-for-TBC](https://github.com/fuba82/Compatibility-Addon-for-TBC) now.  
You need to install it also for this and maybe some of my other AddOns!  
I will add the link the the [Compatibility-Addon-for-TBC](https://github.com/fuba82/Compatibility-Addon-for-TBC) on the Main Page for all my TBC AddOns so all should work without problems!

**Comment by fuba82** [comment]
P.s
would be nice if you can test if it's working with the [Compatibility-Addon-for-TBC](https://github.com/fuba82/Compatibility-Addon-for-TBC)!

**Comment by Macumbafeh** [comment]
I will. I Guess your compatibility Addon is the same than the ElvUI compatibility Addon?
Or do have an other version?

I'll try all your Addons with it. You may add compatibility required for all of them inside the .toc file. 👍

**Comment by fuba82** [comment]
Yeah i think it's the same then ElvUI for TBC uses, hopefully i've not done any changes nut i dont think so! :P
Already added it in the TOC file in required addons, hopyfully i dont forgot any ^^

**Comment by Macumbafeh** [comment]
Hey !
Now i have no errors and i can see the White/Black list
![WoWScrnShot_092320_150813](https://user-images.githubusercontent.com/47739411/94016963-19ab9200-fdaf-11ea-8481-02f8e8f441e6.jpg)

So i tried it, i added Iron ore to the white list
![WoWScrnShot_092320_151017](https://user-images.githubusercontent.com/47739411/94017042-334cd980-fdaf-11ea-84f0-37150a9e0bc3.jpg)

But when i open my bank, it does nothing : /
![WoWScrnShot_092320_151026](https://user-images.githubusercontent.com/47739411/94017063-3fd13200-fdaf-11ea-995d-79f6982b88d7.jpg)

I missunderstood something or?

**Comment by fuba82** [comment]
maybe try to click the "Deposit" button in your Bank window ^^

**Comment by Macumbafeh** [comment]
OMG i didn't see it ! haha i'm noob. i thought it was auto ^^. i'll try again !

**Comment by Macumbafeh** [comment]
Now it works ! thank you ! Just needed to understand how. You don't really need the Whitlist, if there is nothing on Blacklist it will empty everything on your bags !
Thanks !
Now just Molinari-TBC doesn't work at all : /.

**Comment by Macumbafeh** [comment]
I have made some screen if you want to add them to your readme for poeple :) 
![image](https://user-images.githubusercontent.com/47739411/94125733-094ef200-fe57-11ea-95fd-6fcf74a375c4.png)

![image](https://user-images.githubusercontent.com/47739411/94125733-094ef200-fe57-11ea-95fd-6fcf74a375c4.png)
